boards/arm/pinnacle_100_dvk/pinnacle_100_dvk.yaml

```diff
@@ -0,0 +1,23 @@
+identifier: pinnacle_100_dvk
+name: pinnacle_100_dvk
+type: mcu
+arch: arm
+ram: 256
+flash: 1024
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
+  - adc
+  - ble
+  - counter
+  - gpio
+  - i2c
+  - ieee802154
+  - pwm
+  - spi
+  - usb_cdc
+  - usb_device
+  - watchdog
+  - netif:openthread
```

Comment:
If "Pinnacle 100 (HL7800 driver) does not support IPv6" then why is here "netif:openthread" present?